File: launchdarkly/typescript-node/src/PatchTeamsExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.TeamsBetaApi();
apiCaller.setApiKey(api.TeamsBetaApiApiKeys.ApiKey, "YOUR_API_KEY");

const teamsPatchInput = new models.TeamsPatchInput();
teamsPatchInput.instructions =   [
  {
    "kind": "addMembersToTeams",
    "memberIDs": [
      "1234a56b7c89d012345e678f"
    ],
    "teamKeys": [
      "example-team-1",
      "example-team-2"
    ]
  }
];
teamsPatchInput.comment = "Optional comment about the update";

apiCaller.patchTeams(
  teamsPatchInput,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling TeamsBetaApi#patchTeams:");
  console.log(error.body);
});

File: launchdarkly/kotlin/src/main/kotlin/oseg/launchdarkly_examples/PatchTeamsExample.kt
package oseg.launchdarkly_examples

import com.launchdarkly.client.infrastructure.*
import com.launchdarkly.client.apis.*
import com.launchdarkly.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class PatchTeamsExample
{
    fun patchTeams()
    {
        ApiClient.apiKey["ApiKey"] = "YOUR_API_KEY"

        val teamsPatchInput = TeamsPatchInput(
            instructions = Serializer.moshi.adapter<List<Map<String, Any>>>().fromJson("""
                [
                    {
                        "kind": "addMembersToTeams",
                        "memberIDs": [
                            "1234a56b7c89d012345e678f"
                        ],
                        "teamKeys": [
                            "example-team-1",
                            "example-team-2"
                        ]
                    }
                ]
            """)!!,
            comment = "Optional comment about the update",
        )

        try
        {
            val response = TeamsBetaApi().patchTeams(
                teamsPatchInput = teamsPatchInput,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling TeamsBetaApi#patchTeams")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling TeamsBetaApi#patchTeams")
            e.printStackTrace()
        }
    }
}
